R/utils.R

```diff
@@ -7,7 +7,7 @@ set_os_key <- function(api_key) {
 }
 
 
-
+#' @export
```

Comment:
This should still be an internal function. Do you mind changing this to `#' @keywords internal`?

Reply:
I use set_os_key and get_os_key in my non-package code,  ie as a user.  I do not see the functions if remove @export.
Do I misunderstand?
Also,  [r-pkgs version 2](https://r-pkgs.org/namespace.html) :  
"You export functions that you want other people to use."

Reply:
If it is to be exported, please document the function. I personally do not see the need but another exported function cannot hurt. 

Please change to #' @keywords internal or add additional documentation. Thanks!